app-tasks/rf/src/rf/uploads/geotiff/factories.py

```diff
@@ -91,6 +92,14 @@ def create_geotiff_image(self, tif_path, source_uri, scene, filename):
                                     filename=filename, visibility=self.visibility, owner=self.owner)
 
     def create_geotiff_scene(self, tif_path, name):
+        # If this upload is not associated with a project, set the scene's
+        # ingest status to TOBEINGESTED so that scene creation will kick off
+        # an ingest. Otherwise, set the status to NOTINGESTED, so that the status
+        # will be updated when the scene is added to this upload's project
```

Comment:
Oh yeah. I thought "hey that's great I used the geotiff scene creation thing for planet everything's fine" but that's obviously not the case